src/Faithlife.WebRequests/Faithlife.WebRequests.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageReference Include="Faithlife.Json" Version="0.5.0" />
     <PackageReference Include="Faithlife.Utility" Version="0.8.0" />
     <PackageReference Include="JetBrains.Annotations" Version="10.4.0" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="2.1.0" />
```

Comment:
Is there any particular reason to pick 2.1.0 over 2.2.0?

Reply:
2.1 is the LTS version.